src/WebJobs.Script/Rpc/RpcServerInitializationService.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.Abstractions;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public class RpcServerInitializationService : IHostedService
+    {
+        private IRpcServer _rpcServer;
+        private ILogger _logger;
+
+        public RpcServerInitializationService(IRpcServer rpcServer, ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger("Host.RpcServerInitializationService");
+            _rpcServer = rpcServer ?? throw new ArgumentNullException(nameof(rpcServer));            //try
+        }
+
+        public Task StartAsync(CancellationToken cancellationToken)
+        {
+            try
+            {
+                _logger.LogInformation("Initializaing RpcServer");
```

Comment:
Here you say RpcServer, below message you say Grpc Service. Can we make these consistent?